src/button.js

```diff
@@ -1,7 +1,7 @@
 import React, {Component} from 'react'
 import PropTypes from 'prop-types'
 
-import {AUTOCOMPLETE_CONTEXT} from './constants'
+import AUTOCOMPLETE_CONTEXT from './context'
```

Comment:
Yeah, just add an ignore. And you can file an issue on eslint-config-kentcdodds to remove all the dumb rules I have enabled :P (seriously, been planning on disabling many of them recently)